o365enum.py

```diff
@@ -3,65 +3,77 @@
 Office365 User Enumeration script.
 Enumerate valid usernames from Office 365 using the office.com login page.
 
-Author: Quentin Kaiser <quentin@gremwell.com>
-Author: Cameron Geehr @BarrelTit0r
+Author: Quentin Kaiser <quentin@gremwell.com> (original)
+Author: Cameron Geehr @BarrelTit0r (refined and removed login attempt requirement)
```

Comment:
I didn't remove the login attempt requirement, gremwell found the same username enumeration thing as me but I was able to dig a little deeper into the source code for the page and find what each response code actually meant rather than just reporting 1, 5, and 6 all as valid user. Basically just refined the tool

Reply:
Adjusted contributions